### ISSUE TITLE
module 'os' has no attribute 'pathconf'

### ISSUE THREAD

**Issue by Zottelchen** [archivy/archivy#234]
System: Windows 10 Pro 20H2 / Python 3.9.5
When trying to create a new note or add a bookmark, this error appears:
```
ERROR in app: Exception on /notes/new [POST]
Traceback (most recent call last):
  File "c:\python39\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\python39\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\python39\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\python39\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\python39\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\python39\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\python39\lib\site-packages\archivy\routes.py", line 91, in new_note
    note_id = note.insert()
  File "c:\python39\lib\site-packages\archivy\models.py", line 195, in insert
    create(
  File "c:\python39\lib\site-packages\archivy\data.py", line 116, in create
    max_filename_length = os.pathconf(str(data_dir), "PC_NAME_MAX")
AttributeError: module 'os' has no attribute 'pathconf'
```

**Comment by Uzay-G** [comment]
Thanks for reporting this compatibility issue, I fixed it in recent commits and will send a message once I push a new release with the fix in it :).

Should be soon ^^ :smile:

**Comment by Uzay-G** [comment]
Alright this is fixed in 1.3.0 :tada:
